nettoyage.py

```diff
@@ -159,14 +158,11 @@ def manage_montant(df):
     df['montant'] = np.where(df['montant'] <= montant_borne_inf, 0, df['montant'])
     df['montant'] = np.where(df['montant'] >= montant_borne_sup, 0, df['montant'])
 
-    # On applique au df la division
-    df["montant"] = df["montant"] / df["nbTitulairesSurCeMarche"]
-
     # Nettoyage colonnes
-    df['montant'] = np.where(df['montant'] == 0, np.NaN, df['montant'])
+    df['montant'] = np.where(df['montant'].isnull(), 0, df['montant'])
```

Comment:
```suggestion
    df['montant'].fillna(0, inplace=True)
```
cf [Series.fillna](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.fillna.html)